internal/cloud/awscloud/secure-instance.go

```diff
@@ -419,7 +419,6 @@ func (a *AWS) createOrReplaceLT(hostInstanceID, imageID, sgID, iamProfile, keyNa
 				NetworkInterfaceCount: &ec2types.NetworkInterfaceCountRequest{
 					Min: aws.Int32(1),
 				},
-				SpotMaxPricePercentageOverLowestPrice: aws.Int32(200),
```

Comment:
So we expect that the algorithm below does not behave differently or "try harder" if we set a spot price?
Where we could leave the limit here and have another retry where we remove the limit?
(or we do expect it but accept the possibly raised costs?)

Reply:
https://docs.aws.amazon.com/autoscaling/ec2/APIReference/API_InstanceRequirements.html the default behaviour is less strict than what we have now. We already know this because of CI really, they rarely fail to provision spot instances.

Reply:
The question was more, if we get cheaper spot instances when we search first with a limit, then retry without instead of having no limit in the first try. I didn't find infos if this could happen.

Reply:
Right, since we're using `SpotAllocationStrategyPriceCapacityOptimized` we're already not searching for the absolute cheapest option, but rather a mix between cheap and unlikely to get culled. We could of course do retries based on this but I'm also leaning towards just letting AWS's algorithms figure things out.

Reply:
Thanks for your thoughts & replies. Consider this resolved!